_rules/meta-viewport-b4f0c3.md

```diff
@@ -243,15 +243,15 @@ There is no viewport `meta` element.
 </html>
 ```
 
-#### Inapplicable Example 2
+#### Inapplicable Example 4
 
-This viewport `meta` element does not have a `content` attribute.
+This viewport `meta` element does not prevent user scaling because it does not specify the `maximum-scale` nor `user-scalable` values.
```

Comment:
What's the difference for Inapplicable Example 3?

Reply:
I think @WilcoFiers asked me to include these changes? I honestly don't remember changing the examples, but rather just moving one of them from passed to N/A.

Reply:
Talked to Carlos. The point is that both example 3 and 4 have a content attribute, but he said he doesn't mind it. The two examples are different enough. We're good to move forward with this.